src/components/dialogs/parameters/dynamicsimulation/curve/dialog/equipment-filter.js

```diff
@@ -35,8 +35,29 @@ import {
 export const CURVE_EQUIPMENT_TYPES = [
     EQUIPMENT_TYPES.GENERATOR,
     EQUIPMENT_TYPES.LOAD,
+    EQUIPMENT_TYPES.BUS,
+    EQUIPMENT_TYPES.BUSBAR_SECTION,
 ];
 
+// this function is used to redirect an equipment type to the referenced equipment type which is used in the default model.
+export const getReferencedEquipmentTypeForModel = (equipmentType) => {
+    // particular case, BUSBAR_SECTION and BUS use the same default model for Bus
+    return equipmentType === EQUIPMENT_TYPES.BUSBAR_SECTION
+        ? EQUIPMENT_TYPES.BUS
+        : equipmentType;
+};
+
+// this function is used to provide some basic values of expert filter for some particular cases
+const getExpertFilterBasicFieldsForType = (equipmentType) => {
```

Comment:
I know you wanted to do something generic, but it's not better to call it like that and eventually create other methods if necessary ?
```suggestion
const getTopologyKindIfNecessary = (equipmentType) => {
```